src/search/find-visits.js

```diff
@@ -75,42 +77,40 @@ export function findVisitsToPages({pagesResult}) {
      * Raj Pratim Bhattacharya gmail rajpratim1234@gmail.com
      */
     
-    var comp = new Date();
-    var sDate = comp.getTime() - 100*24*60*60*1000 //100 days old search
-    var eDate =  comp.getTime()
-    
+
+    var current_time = new moment().valueOf();
+    var startDate = current_time - 100*24*60*60*1000; //100 days old search
```

Comment:
@obsidianart  I am afraid , I coudn't get you , can you refer where I have used moment-days previously.